### PR TITLE
Update docs to match robotpy/examples/pull/12

### PR DESCRIPTION
**Pull request by WardBenjamin**
This changes the getting-started example to match the example PR (https://github.com/robotpy/examples/pull/12) (aka switches to using a Timer for autonomous timing).

### CODE REVIEW

File: guide/anatomy.rst [hunk 1]
```diff
@@ -223,29 +223,26 @@ below, taken from one of the samples in our github repository::
...
+            self.timer.start()
+
         def autonomousPeriodic(self):
             """This function is called periodically during autonomous."""
-    
-            # Check if we've completed 100 loops (approximately 2 seconds)
-            if self.auto_loop_counter < 100:
+
+            # Drive for two seconds
+            if self.timer.get() < 2.0:
```
Comment (twinters007) [inline guide/anatomy.rst]: I like using `timer.hasPeriodPassed` because that'll reset it so you can chain these. Otherwise you gotta do quick math

Comment (WardBenjamin) [reply guide/anatomy.rst]: I'd prefer to keep it as-is to match upstream, but it can be changed if others agree.

Comment (twinters007) [reply guide/anatomy.rst]: Oh is this supposed to match the screensteps or something?

Comment (WardBenjamin) [reply guide/anatomy.rst]: It matches the example in the Eclipse plugins

Comment (twinters007) [reply guide/anatomy.rst]: ah

Comment (auscompgeek) [reply guide/anatomy.rst]: Not the Eclipse plugins, the examples haven't lived there for a year now.

Comment (WardBenjamin) [reply guide/anatomy.rst]: According to https://wpilib.screenstepslive.com/s/currentCS/m/java/l/145307-creating-your-benchtop-test-program, they do. This code also matches code on that ScreenSteps page.

Comment (auscompgeek) [reply guide/anatomy.rst]: The examples are included with the Eclipse plugins, yes. They aren't maintained there any more — they live in the allwpilib repo now, so you should be basing any work off there.

**Comment by WardBenjamin** [comment]
This should probably be merged as-is, since the example was just merged. Looks good compared to the examples in allwpilib as well.

**Pull request merged**